docs/sources/docker-hub/official_repos.md

```diff
@@ -1,189 +1,110 @@
+feedback, contribute code, suggest process changes, or even propose a new
+official repository.
+
+## Should I use Official Repositories?
+
+New Docker users are encouraged to use the Official Repositories in their
+projects. These repositories have clear documentation, promote best practices,
+and are designed for the most common use cases. Advanced users are encouraged to
+review the Official Repositories as part of their `Dockerfile` learning process.
+
+A common rationale for diverging from Official Repositories is is to optimize
+for image size. For instance, many of the programming language stack images
+contain a complete build toolchain to support installation of modules that
+depend on optimized code. An advanced user could build a custom image with just
+the necessary pre-compiled libraries to save space and reduce attack vectors in
+production.
+
+A number of language stacks such as
+[`python`](https://registry.hub.docker.com/_/python/) and
+[`ruby`](https://registry.hub.docker.com/_/ruby/) have `-slim` tag variants
+designed to fill the need for optimization.  Even when these "slim" variants are
+insufficient, it is still recommended to inherit from an official base OS image
+to leverage the ongoing maintenance work, rather than duplicating these efforts.
+
+## How can I get involved?
+
+All Official Repositories contain a **User Feedback** section in their
+documentation which covers the details for that specific repository. In most
+cases, the GitHub repository which contains the Dockerfiles for an official repo
+also has an active issue tracker. General feedback and support questions should
+be directed to `#docker-library` on Freenode IRC.
+
+## How do I create a new official repository?
+
+This page is intended as a high level overview of the process. An Official
+Repository starts out as a proposal in the form of a set of GitHub pull
+requests.  You'll find detailed and objective proposal requirements in the
+following GitHub repositories:
+
+* [docker-library/official-images](https://github.com/docker-library/official-images)
+
+* [docker-library/docs](https://github.com/docker-library/docs)
+
+Carefully review these repositories before submitting a proposal. The Official
+Repositories team, with help from community contributors, formally review each
+proposal and provide feedback to the author. The initial review process usually
+requires a bit of back and forth, but the end result will either be a merged
+pull request, or a rejection.
+
+There are also subjective considerations during the review process. These
+subjective concerns boil down to the basic question: "is this image generally
+useful?"  For example, the [`python`](https://registry.hub.docker.com/_/python/)
+repo is "generally useful" to the large Python developer community, whereas an
+obscure text adventure game written in Python last week is not. If you are
+unclear about your particular use case, please reach out before committing to a
```

Comment:
Removed the sentence as it wasn't really adding much value.  We've already directed the reader to engage on IRC for general questions/feedback.